### DISCUSSION TITLE
Notifications in V#

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#7507] [Help]
In the version 2 of the project i used to enable the notifications with this configuration:

'notifications' => [
            'vertical_alignment' => 'top',
            'alignment' => 'right',
        ],

I dont know how to do it in the V3. Any Advice?

Thanks in advanced.

**Comment by pboivin** [comment]
Are you trying to customize the default style of the notifications?

**Reply by jcgmalave** [reply]
Not really, i followed the steps and the section of notifications doesnt appear in the top menu.

**Reply by pboivin** [reply]
If you're in a Panel, you should be able to send notifications like this:
```php
use Filament\Notifications\Notification;

Notification::make()
    ->title('Hello')
    ->success()
    ->send();
```

**Comment by sumardi** [comment]
In service provider

```php
use Filament\Notifications\Livewire\Notifications;
use Filament\Support\Enums\Alignment;
use Filament\Support\Enums\VerticalAlignment;

Notifications::alignment(Alignment::Center);
Notifications::verticalAlignment(VerticalAlignment::Start);
```

**Reply by jcgmalave** [reply]
I still don't see the notifications, even though they are being saved in the database :(

**Reply by pboivin** [reply]
I feel like we're missing some context to be able to help you... Can you share the code you have been trying?

**Reply by jcgmalave** [reply]
my config/filament looks like this:

'broadcasting' => [

         'echo' => [
             'broadcaster' => 'pusher',
             'key' => env('VITE_PUSHER_APP_KEY'),
             'cluster' => env('VITE_PUSHER_APP_CLUSTER'),
             'forceTLS' => true,
         ],

    ],

my .env file looks like this:

PUSHER_APP_ID=
PUSHER_APP_KEY=
PUSHER_APP_SECRET=
PUSHER_HOST=
PUSHER_PORT=443
PUSHER_SCHEME=https
PUSHER_APP_CLUSTER=mt1

VITE_APP_NAME="${APP_NAME}"
VITE_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
VITE_PUSHER_HOST="${PUSHER_HOST}"
VITE_PUSHER_PORT="${PUSHER_PORT}"
VITE_PUSHER_SCHEME="${PUSHER_SCHEME}"
VITE_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"

**Reply by pboivin** [reply]
So this is a question about Broadcast notifications using Pusher?

**Reply by jcgmalave** [reply]
Yes, the Broadcast notifications are not showing
